app-backend/migrations/src_migrations/main/scala/54.scala

```diff
@@ -0,0 +1,15 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M54 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(54)(List(
+    sqlu"""
+    CREATE TABLE organization_features (
+      organization UUID NOT NULL UNIQUE,
+      feature_flags UUID[],
```

Comment:
These are the flags we have now in staging:

```json
[
  {
    "id": "68144a5a-bf71-4925-b2f6-94893333c88d",
    "key": "profile-org-edit",
    "active": false,
    "name": "Profile Organization Edit",
    "description": "Profile Organization Edit"
  },
  {
    "id": "76ffc49d-f92f-4f4f-99ad-40e8b1ff529f",
    "key": "make-source-histogram",
    "active": false,
    "name": "Make source histogram",
    "description": "Make source histogram"
  },
  {
    "id": "515c2eb5-7255-4c31-aef8-8423ba15d6c7",
    "key": "market-search",
    "active": false,
    "name": "Market Search",
    "description": "Market Search"
  },
  {
    "id": "99e38005-0ae5-41cd-bbea-330bda92499a",
    "key": "display-histogram",
    "active": false,
    "name": "Display Histogram",
    "description": "Display Histogram"
  },
  {
    "id": "a75d427f-576d-4850-96c3-895bbf384098",
    "key": "tools-ui",
    "active": true,
    "name": "Enable Tools UI",
    "description": "Enable Tools UI"
  }
]```

I think we have to also have an enable/disable value in this list in order to be able to distinguish between making a flag inactive or active for an organization. For instance, worst case here would be an organization that has flags the flip the opposite of the environment defaults.